src/components/containers/StudentDashboard/StudentWorkshopRegistration.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import UserContext from "../../../UserContext";
+import { ISession } from "../../../logic/domains/sessionDetails.domain";
+import getStudentWorkshops from "../../../logic/functions/getStudentWorkshops";
+import MaterialTable from "material-table";
+import { RouteComponentProps } from "react-router";
+import { Link, withRouter } from "react-router-dom";
+
+type StudentWorkShopProps = RouteComponentProps<any> & {};
+
+let workshopData: any;
+
+const StudentWorkshopRegistration: React.FunctionComponent<StudentWorkShopProps> = props => {
+	const [sessions, setSessions] = React.useState<Array<ISession>>([]);
```

Comment:
this isn't being used?? 
Also should it be of type `Array<ISession>` or `Array<IWorkshop>`???